.pre-commit-config.yaml

```
@@ -25,15 +28,24 @@ repos:
     hooks:
       - id: black
         exclude: '(?:extern/.*)'
-  - repo: local
+  - repo: https://github.com/PyCQA/pydocstyle
+    rev: '6.1.1'
     hooks:
-      - id: flake8
-        name: flake8
-        entry: flake8
-        language: python
-        types: [python]
+      - id: pydocstyle
+        exclude: '(?:extern/.*|tests/.*|setup.py|conf.py)'
+  - repo: https://gitlab.com/PyCQA/flake8
```

The repo has moved from GitLab to to GitHub.
```suggestion
  - repo: https://github.com/PyCQA/flake8
```

---

Whoops missed this one.